css/shortcode-ui-richtext.css

```diff
@@ -0,0 +1,10 @@
+/* Make sure any popup boxes appear */
+div.mce-inline-toolbar-grp.mce-arrow-up,
+div.mce-floatpanel[style]{
+    z-index: 999999!important;
```

Comment:
I'll try to stay away from !important rules, and see whether a stronger selector would suffice here.